controlplane/src/core/util.ts

```diff
@@ -69,6 +69,10 @@ export function isValidLabels(labels: Label[]): boolean {
   return true;
 }
 
+function isValidLabel(value: string): boolean {
```

Comment:
This name indicates it tests both key and value. I would just remove this function seeing its usage.